src/state/lists/hooks.ts

```diff
@@ -55,11 +56,11 @@ function listToTokenMap(list: TokenList): TokenAddressMap {
 
   const map = list.tokens.reduce<TokenAddressMapWriteable>(
     (tokenMap, tokenInfo) => {
-      if (tokenMap[tokenInfo.chainId][tokenInfo.address] !== undefined) {
+      if (tokenMap[tokenInfo.chainId][getFormattedAddress(tokenInfo.address)] !== undefined) {
```

Comment:
Too long expression here. Please split them into parts

Reply:
how would you suggest?

Reply:
```ts
  const map = list.tokens.reduce<TokenAddressMapWriteable>((tokenMap, tokenInfo) => {
    const tokenAddress = getFormattedAddress(tokenInfo.address)

    if (tokenMap[tokenInfo.chainId][tokenAddress] !== undefined) {
      return tokenMap
    }

    const token = new WrappedTokenInfo(tokenInfo, list)
    tokenMap[tokenInfo.chainId][tokenAddress] = token
    return tokenMap
  }, EMPTY_LIST())
```

or even better, you can take chainId out of tokenInfo

Reply:
I removed spread in EMPTY_LIST() as it's not needed

Reply:
done